Added form action attribute to thread.php file

DrDoomsday opened the pull request:
This fixes issues #3

DrDoomsday commented:
DrDoomsday/forum#2

paulstatezny commented on public/thread.php:
```diff
@@ -1,7 +1,7 @@
 
 
-<form>
+<form action="post_reply.php" method="post">
```
So you kind of made this file into a "post thread" **and** a "post reply" page. Which doesn't really make sense. But either way, this action belongs in the form for posting a reply, which is on line 19.

paulstatezny commented:
:star:

DrDoomsday commented:
@paulstatezny this is ready for review.

paulstatezny commented on public/thread.php:
```diff
@@ -1,19 +1,16 @@
-<form>
+<form action="post_reply.php" method="post">
 	Title:
 	<input type="text">
```
You left this input here from the "make thread" form. It doesn't belong.

paulstatezny replied:
In fact the `<form>` doesn't belong either.

paulstatezny commented on public/make_thread.php:
```diff
@@ -0,0 +1,17 @@
+
+		<textarea>
+			Enter text here...
+		</textarea>
```
Can we fix the indentation here?

paulstatezny commented on public/thread.php:
```diff
@@ -1,19 +1,16 @@
+<div class="thread-post">
+	This will be where the thread post will go. The first thread post from the thread author will be the first post on the page.
+</div>
 	<br>
```
I would un-indent this `<br>` too.

paulstatezny commented on public/thread.php:
```diff
@@ -1,19 +1,12 @@
+<div class="thread-post">
+	This will be where the thread post will go. The first thread post from the thread author will be the first post on the page.
+</div>
 	<br>
```
I would un-indent this `<br>` too.

paulstatezny commented:
This PR looks complete -- if you can fix the indentation I'll merge it.

DrDoomsday commented:
@paulstatezny I fixed the indentation.

DrDoomsday commented:
@paulstatezny I used your code from jsfiddle. I like the way the format looks. I deleted thread.php and thread.html because we aren't using them for anything (as far as I could tell). I made a file called view_thread.php so I could keep it clear in my head which was viewing threads and which was making a thread. I added the code to the header.php file that applies to make_thread.php and view_thread.php

DrDoomsday commented:
@paulstatezny I just realized I was pushing ALL changes I was making to this pull request. Sorry about that. How do I push to the master from git bash rather than pushing specifically to this branch?

paulstatezny commented:
@DrDoomsday Ok sounds good. You're getting more comfortable with this -- good!

No worries about making the changes here. It would have been good to do that in a separate pull request, but since you already pushed the commits we'll just go with it.

paulstatezny commented:
About the indentation: I think you misunderstood, there were just two `<br>` tags that needed indented one to the right. But it's all good.

I'm going to fix some indentation on this new file and merge. You can look at my commit to see what needed changed.

paulstatezny commented:
Also, good catch wiring up that button on the main page.

paulstatezny commented:
Note, I deleted `public/usethistocheckchanges.html`.

paulstatezny commented:
Actually, I won't merge this, since I made a change. I'll let you (skim) review my code changes and merge when you're ready.
